special-use/special-use-suds-mapping.json

```diff
@@ -6,97 +6,97 @@
+          "mailingAddress2": { "type": "string", "basic-api-field" : "ADDRESS_2"},
+          "mailingCity": { "type": "string", "basic-api-field" : "CITY_NAME"},
+          "mailingState": { "type": "string", "basic-api-field" : "STATE_CODE"},
+          "mailingZIP": { "type": "integer", "basic-api-field": "POSTAL_CODE"},
       		"organizationName": {
       			"description": "The name of the group, company, organization or event applying",
       			"type": "integer",
-            "suds-application": "Contact",
-            "suds-field": "Contact Name"
       		},
           "website": { "type": "string", "suds-table": "Contact", "suds-field": "Remarks" },
-          "orgType": { "type": { "enum": [ "Individual"  , "Corporation"  , "Limited Liability Company"  , "Partnership or Association"  , "State Government or Agency"  , "Local Government or Agency"  , "Nonprofit" ] }, "suds-table": "Contact Type", "suds-field": "Remarks" },
-        "required": ["firstName","lastName","dayPhone","emailAddress","mailingAddress","mailingCity","mailingZIP","mailingState","organizationName"]
+          "orgType": { "type": { "enum": [ "Individual"  , "Corporation"  , "Limited Liability Company"  , "Partnership or Association"  , "State Government or Agency"  , "Local Government or Agency"  , "Nonprofit" ] }, "basic-api-field" : "ORG_TYPE, ORG_CODE" },
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "COUNTRY_NAME", "value" : "United States of America"},
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "CONT_NAME", "note" : "Should be either the organizationName or the first/last name"},
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "CONTACT_TYPE", "note" : "If organizationName is filled, then should be ORGANIZATION. If not PERSON."},
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "ADDRESS_TYPE", "value" : "Main"},
+
+        "required": ["firstName","lastName","dayPhone","emailAddress","mailingAddress","mailingCity","mailingZIP","mailingState"]
       },
       "phoneNumber": {
         "type": "object",
         "properties": {
-          "internationalCode": { "type": "integer", "suds-application": "Contact", "suds-field": "International Code"},
-          "areaCode": { "type": "integer", "suds-application": "Contact", "suds-field": "Area Code"},
-          "number": { "type": "integer", "suds-application": "Contact", "suds-field": "Phone Number"},
-          "extension": { "type": "integer", "suds-application": "Contact", "suds-field": "Extension"},
-          "type": { "suds-application": "Contact", "suds-field": "Phone No. Type", "enum" : ["non-commercial","temp-outfitter-guide"]},
+          "areaCode": { "type": "integer", "basic-api-field": "AREA_CODE"},
+          "number": { "type": "integer", "basic-api-field": "PHONE_NUMBER"},
+          "extension": { "type": "integer", "basic-api-field": "EXTENSION"},
```

Comment:
It's a field on the FS form, so I put it in here, but it's not required. Is that OK?